### PR TITLE
[12.0][ADD] web_widget_google_address_autocomplete

### PR DESCRIPTION
**Pull request by murtuzasaleh**
Google Address Form Autocomplete
============================
Enable Google Address form autocomplete on Website sale customer form.

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
@dreispt @pedrobaeza @wolfhall @max3903

**Comment by dreispt** [comment]
From the description I can't tell what this module is doing.
I thought this was adding a Form widget with address autocompletion capabilities.
Can you add an USAGE file explaining?
It would also be nice if you could share an image.

File: web_widget_google_address_autocomplete/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2019, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Google Address Form Autocomplete',
```
Comment (dreispt) [inline web_widget_google_address_autocomplete/__manifest__.py:4]:
```suggestion
    'name': 'Google Address Form Autocomplete',
    'summary': 'Enable Google Address form autocomplete on Website sale customer form',
```

**Review by dreispt — COMMENTED** [review]
LGTM, please make Mr. Travis happy.

**Comment by murtuzasaleh** [comment]
Depend on #223

**Review by dreispt — APPROVED** [review]

**Review by yvaucher — CHANGES_REQUESTED** [review]
Please give some love to the pylint warnings. Adding an other module with that much warnings makes the travis log unreadable.

To quote pylint-odoo
```
Your code has been rated at -1.43/10
```

Lint might be nitpicking on few thing, but if you can fix the missing trailing coma plus split the lines that are too long we could get rid of most of them.

**Comment by dreispt** [comment]
@murtuzasaleh Can you run a linter on the code, such as Black, please?

**Comment by dreispt** [comment]
Sorry, I see now the issue is not pylint, but rather javascript-lint.
Is there any linter available to do this?

**Pull request closed without merging**